src/javascripts/modules/pillbar.js

```diff
@@ -0,0 +1,28 @@
+export default class Pillbar {
+  constructor(el) {
+    this.el = el
+    this.ticking = false
+    this.lastScrollY = 0
+
+    window.addEventListener('scroll', () => this.handleScroll())
+  }
+
+  handleScroll() {
+    this.lastScrollY = window.scrollY
+    if (!this.ticking) {
+      window.requestAnimationFrame(() => {
+        this.togglePillbar()
+      })
+    }
+    this.ticking = true
+  }
+
+  togglePillbar() {
+    this.ticking = false
+    if (this.lastScrollY > 420) {
```

Comment:
is there a better way to set this number that isn't so heavily determined by the CSS? it may be better to gather the height of the elements that you don't want the scroll to activate for within JS, so that if the height on things ever changes (especially for responsive designs) you aren't stuck having it break for no reason.